pre-docs/bolt_options.md

```diff
@@ -97,6 +97,8 @@ transport for each node. To override the default transport, specify the
 protocol on a per-host basis, such as `bolt command run facter --nodes
 win1,ssh://linux --transport winrm`
 
+If ```localhost``` is passed to ```--nodes``` when invoking Bolt on a non-windows OS the `local` transport is used automatically. This behaviour can be avoided by prepending the target with the desired transport (ex: ```ssh://localhost```).
```

Comment:
Most of these examples could use single backticks rather than triple. I'm surprised it formats well.